message-index/messages/GHC-20125/example1/after/Error.hs

```diff
@@ -0,0 +1,7 @@
+{-# OPTIONS_GHC -Wmissing-fields #-}
+module Error where
+
+data Rec = Rec { a :: Int, b :: String, c :: Bool }
+
+x :: Rec
+x = Rec { a = 1, b = "two", c = True }
```

Comment:
This file doesn't end in a newline, but its counterpart does. This might be an issue when we get `diff` showing on the example pages.

Reply:
Done in 10b0051